ensembl/htdocs/ssi/species/Cyprinus_carpio_carpio_annotation.html

```diff
@@ -0,0 +1,6 @@
+</p>
```

Comment:
Could you remove this stray "close paragraph" tag?

Reply:
We shouldn't have all these merge commits I think ?